app/Helpers/url_helper.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+if (! function_exists('url_is'))
```

Comment:
I really like this function! It might be good to a dd it upstream in CI4 instead? I see a lot of good use for it outside of this project.

Reply:
I have definitely planned on porting it there, I just hadn't time to put together tests for it :)